README.md

```diff
@@ -66,8 +66,9 @@ const fastifyPassport = new Authenticator()
 server.register(fastifyCookie)
 server.register(fastifySession, { secret: 'secret with minimum length of 32 characters' })
 
-// initialize fastify-passport
+// initialize fastify-passport and connect it to the secure-session storage. Note: both of these plugins are mandatory.
 server.register(fastifyPassport.initialize())
+server.register(fastifyPassport.secureSession())
```

Comment:
Would you mind adding a test case here https://github.com/fastify/fastify-passport/blob/main/test/passport.test.ts ?

Reply:
Not really sure what should be tested? The full example in the readme?